profile-widget/diveeventitem.cpp

```diff
@@ -152,17 +152,17 @@ void DiveEventItem::setupToolTipString()
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
-			name += QString(" (cyl %1)").arg(internalEvent->gas.index+1);
+			name += tr(" (cyl. %1)").arg(internalEvent->gas.index + 1);
 	} else if (value) {
-		if (type == SAMPLE_EVENT_PO2 && name == "SP change") {
+		if (type == SAMPLE_EVENT_PO2 && (QString) internalEvent->name == "SP change") {
```

Comment:
One needs to compare against the original name string here, not the translated one!
Question: I found that I need the cast to QString here to make it work. Is this correct or is there a better way?